src/database/migrations/20210907231231_create_table_documento_medico.js

```diff
@@ -0,0 +1,19 @@
+exports.up = async function (knex) {
+    return await knex.schema.createTable('documento_medico', table => {
+        table.increments('id').primary();
+        table.integer('id_paciente').references('paciente.id').notNullable().onDelete('CASCADE');
+        table.text('tipo').notNullable();
+        table.text('descricao').notNullable();
+        table.timestamp('dt_hr_emissao').nullable();
+        table.date('dt_validade').nullable();
+        table.text('status').notNullable();
+
+        table.timestamp('created_at').defaultTo(knex.fn.now()).nullable();
+        table.timestamp('updated_at').defaultTo(knex.fn.now()).nullable();
```

Comment:
Aqui seria uma situação semelhante à Anotação, pois as Anotações e Documentos Médicos não podem ser alterados, e o atributo `dt_hr_emissao` na linha 7 tem o mesmo papel do `created_at` na linha 11.

```suggestion
        table.timestamp('dt_hr_emissao').defaultTo(knex.fn.now()).nullable();
        table.date('dt_validade').nullable();
        table.text('status').notNullable();
```